### PR TITLE
doc: refactor machine-learning-overview.md

### PR DESCRIPTION
**Pull request by arjunsajeev**

### CODE REVIEW

**Comment by yorkie** [comment]
@arjunsajeev Thanks for your contribution, and may I ask you to tell the details why this PR does this change?

**Comment by arjunsajeev** [comment]
@yorkie This PR just some updates some of the variable names and rectifies the type assignment. I think these changes improve readability.

**Comment by yorkie** [comment]
> @yorkie This PR just some updates some of the variable names and rectifies the type assignment. I think these changes improve readability.

Ok, thanks. And line 28-29 in this tutorial needs to be updated, and may I ask you to update these names in [our Chinese tutorial](https://github.com/alibaba/pipcook/blob/master/docs/zh-cn/tutorials/machine-learning-overview.md)?

File: docs/zh-cn/tutorials/machine-learning-overview.md [hunk 1]
```diff
@@ -6,28 +6,28 @@
 
 一般来说，一个学习问题就是将 N 个样本集数据输入，然后输出与输入相关联对应的结果，下面的例子将展示，如何教会一个程序学会 Node.js 书籍和售价的关系：
 
-```js
-const model = Record<string, number>;
-const learn = (book: string, price: number) => model[book] = price;
-const how_much = (book: string) => model[book];
+```ts
+const BookPriceModel: Record<string, number> = {}
```
Comment (yorkie) [inline docs/zh-cn/tutorials/machine-learning-overview.md:10]:
```suggestion
const BookPriceModel: Record<string, number> = {};
```

File: docs/tutorials/machine-learning-overview.md [hunk 2]
```diff
@@ -6,28 +6,28 @@ From this article, we will introduce what is machine learning and how to complet
 
 In general, a learning problem considers a set of n samples of data and then tries to returns the corresponding result by given input. The following example shows how we can teach a program about Node.js books and prices:
 
-```js
-const model = Record<string, number>;
-const learn = (book: string, price: number) => model[book] = price;
-const how_much = (book: string) => model[book];
+```ts
+const BookPriceModel: Record<string, number> = {}
```
Comment (yorkie) [inline docs/tutorials/machine-learning-overview.md:10]:
```suggestion
const BookPriceModel: Record<string, number> = {};
```

**Comment by yorkie** [comment]
@arjunsajeev found small nits :)

**Comment by rickycao-qy** [comment]
LGTM

**Pull request merged**